src/components/timer/TimerController.cpp

```diff
@@ -0,0 +1,60 @@
+//
+// Created by florian on 16.05.21.
+//
+
+#include "TimerController.h"
+#include "systemtask/SystemTask.h"
+#include "app_timer.h"
+#include "task.h"
+
+using namespace Pinetime::Controllers;
+
+
+APP_TIMER_DEF(timerAppTimer);
+
+
+TimerController::TimerController(System::SystemTask& systemTask) : systemTask {systemTask} {
+}
+
+
```

Comment:
Can you please make sure that you autoformat the code with clang-format?

(If you use CLion you can enable integration in the settings)

Reply:
this part looks exactly the same after reformating